src/Components/Pages/TransferPage.tsx

```diff
@@ -360,7 +360,7 @@ const TransferPage = () => {
             return parseFloat(value) <= tokens[preflightDetails.token].balance;
           } else {
             return (
-              parseFloat(value + (bridgeFee || 0)) <=
+              parseFloat(value) + (bridgeFee || 0) <=
```

Comment:
Why?

Reply:
The value is returned as a string, so it concatenates with the bridge value. To overcome this issue, I have converted the type of value and then added to bridge fee.